### PR TITLE
Enhance ETH wrap styling.

### PR DESCRIPTION
**Pull request by biocom**
Enhances styling of the ETH wrap component created by @W3stside :
Screen Shot 2021-04-22 at 17 23 24
Screen Shot 2021-04-22 at 17 19 07

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]

File: src/custom/components/swap/EthWethWrap/WrapCard.tsx [hunk 1]
```diff
@@ -27,36 +18,49 @@ const WrapCardWrapper = styled.div`
...
+    right: 0;
+    margin: auto;
     background: ${({ theme }) => theme.white};
-    width: 30px;
-    height: 30px;
-    padding: 5px;
+    width: 24px;
+    height: 24px;
+    border-radius: 24px;
+      padding: 3px;
```
Comment (W3stside) [inline src/custom/components/swap/EthWethWrap/WrapCard.tsx:63]: indent here

File: src/custom/components/swap/EthWethWrap/index.tsx [hunk 2]
```diff
@@ -137,9 +134,7 @@ export default function EthWethWrap({ account, native, userInput, wrapped, wrapC
       <WarningWrapper>
         <AlertTriangle size={25} />
         <div>
-          <span>
-            To sell {nativeSymbol}, first wrap or switch to {wrappedSymbol}
-          </span>
+          Wrap your {nativeSymbol} first or switch to {wrappedSymbol}!
```
Comment (W3stside) [inline src/custom/components/swap/EthWethWrap/index.tsx:137]: not sure the text should be changed in this PR.

I don't like the `!` it's too intense, i would just remove it @anxolin @alfetopito @annamsgeorge

**Comment by W3stside** [comment]
Approving verbally, just minor comments

File: src/custom/components/swap/EthWethWrap/index.tsx (hunk 2 above)
Comment (biocom) [reply src/custom/components/swap/EthWethWrap/index.tsx:137]: Was changed, as the text should be bigger than it was. As a result, trimmed the text down in order to achieve that.

Comment (biocom) [reply src/custom/components/swap/EthWethWrap/index.tsx:137]: Can remove the ! if too intense. Personally good with it.

**Comment by biocom** [comment]
Last commit centers the error message:
Screen Shot 2021-04-23 at 11 05 54

**Review by anxolin — COMMENTED** [review]
soft APPROVE. Leaving @W3stside to do the final approve (so mergfy does his job)

**Review by W3stside — APPROVED** [review]

**Comment by W3stside** [comment]
@Mergifyio update

**Review by anxolin — APPROVED** [review]

**Pull request merged**